src/Controls/tests/TestCases.HostApp/Issues/Issue24061.xaml

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue24061"
+             xmlns:ns="clr-namespace:Maui.Controls.Sample.Issues">
+    <ScrollView Orientation="Horizontal"
```

Comment:
Could be nice if can include for example, a Button, and tapping it add/remove the gradient background. In this way, we can validate changes at runtime.

Reply:
Sure!

Reply:
Done